client/core/core.go

```diff
@@ -1466,11 +1507,11 @@ func (c *Core) resolveActiveTrades(crypter encrypt.Crypter) (loaded int) {
 			c.notify(newOrderNote("Order load failure", details, db.ErrorLevel, nil))
 		}
 		if len(ready) > 0 {
-			locks, err := c.resumeTrades(dc, ready)
-			if err != nil {
-				details := fmt.Sprintf("Some active orders failed to resume: %v", err)
-				c.notify(newOrderNote("Order resumption error", details, db.ErrorLevel, nil))
-			}
+			locks := c.resumeTrades(dc, ready)
+			// if err != nil {
+			// 	details := fmt.Sprintf("Some active orders failed to resume: %v", err)
+			// 	c.notify(newOrderNote("Order resumption error", details, db.ErrorLevel, nil))
+			// }
```

Comment:
Pretty sure this was a relic from when `resumeTrades` could actually return a non-nil error.